backend/src/main.py

```diff
@@ -308,19 +308,24 @@ def __init__(self, user_name, action, base_m, message):
 
     async def callback(self, interaction: nextcord.Interaction):
         # Get the entered amount
-        amount = self.amount_input.value
+        data = self.amount_input.value
 
         #convert amount to numbers
-        try:
-            amount = int(amount)
-        except:
-            amount = 0
+        amount = abs(int(data)) if data.isdigit() else 0
```

Comment:
Is **abs** the correct way to handle negative number ?
Should we handle negative numbers with an error instead ?